src/tests/unit/tests/DetailsView/components/fluent-side-nav.test.tsx

```diff
@@ -40,4 +47,26 @@ describe(FluentSideNav, () => {
 
         expect(wrapper.getElement()).toMatchSnapshot();
     });
+
+    test('dismiss side nav', () => {
+        tabStoreData = {
+            isClosed: false,
+        } as TabStoreData;
+
+        const setSideNavOpenMock = Mock.ofInstance((isOpen: boolean) => {});
+        setSideNavOpenMock.setup(sm => sm(false)).verifiable(Times.once());
+        props = {
+            tabStoreData,
+            isSideNavOpen: false,
```

Comment:
Is the side nav is not open when we call dismiss on it?

Reply:
yes.